tests/storage/storage.go

```diff
@@ -80,7 +80,7 @@ const (
 
 	// Without cloud init user data Cirros takes long time to boot,
 	// so provide this dummy data to make it boot faster
-	cirrosUserData = "#!/bin/bash\necho 'hello'\n"
+	cirrosUserData = libvmi.BashHelloScript
```

Comment:
Can you inline it into uses of `cirrosUserData`?

Reply:
Dropped `cirrosUserData`